cmd/ocm/gcp/update-wif-config.go

```diff
@@ -197,6 +197,42 @@ func updateWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) e
 			"or contact Red Hat support.", wifConfig.ID())
 	}
 
+	// If the WIF pool is hosted in a different project than the main project, we need to verify the pool usage
+	if wifConfig.Gcp().FederatedProjectId() != wifConfig.Gcp().ProjectId() {
```

Comment:
Let's only run this check if the wif-config was updated on this update call. Also, let's add a flag to skip this check.
This initial check should therefore be:
`if wifProjectUpdated() && !skipProjectVerfication()`

Reply:
Also, please move the verification logic into a separate helper. This is to reduce the size of this method and for potential reuse of this functionality.

Reply:
`--skip-cross-project-verification` - Bypasses check when updating the federated project